ndc-client/src/apis/default_api.rs

```diff
@@ -326,3 +333,38 @@ mod utils {
         });
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
If there's really no library for this function then we should probably make one, and move the tests there, because this seems out of scope for `ndc-spec`. I think a `url-utils` crate would be a nice little Rust OSS contribution, but I feel like something ought to exist.

Reply:
(we can do this later, of course)